src/backend/distributed/executor/multi_executor.c

```diff
@@ -131,16 +152,16 @@ CitusExecutorRun(QueryDesc *queryDesc,
 				 ScanDirection direction, uint64 count, bool execute_once)
 {
 	DestReceiver *dest = queryDesc->dest;
-	int originalLevel = FunctionCallLevel;
+	int originalLevel = SPILevel;
 
 	if (dest->mydest == DestSPI)
 	{
 		/*
-		 * If the query runs via SPI, we assume we're in a function call
+		 * If the query runs via SPI, we assume we're in a function call.
```

Comment:
This period doesn't make sense when the following line begins with 'and', comma?